### PR TITLE
Add Prometheus logo in react UI

### LINKED ISSUE #7783: Add the Prometheus logo to the react ui
## Proposal
**Use case. Why is this important?**

I think that except in the favicon, the prometheus logo is nowhere to be seen in the UI.

I'd like to see it in the navbar:

temp

Here is my test code:

<details>
```
<a class="pt-0 navbar-brand" href="/new/graph" aria-current="page"><img src="https://prometheus.io/assets/prometheus_logo_grey.svg" style="vertical-align: text-top;height: 1.4em;padding-right:.2em"> Prometheus</a>
```
</details>

### PR DESCRIPTION
**Pull request by roidelapluie**
Fixes #7783

Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: web/ui/react-app/src/Navbar.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ const Navigation: FC<PathPrefixProps & NavbarProps> = ({ pathPrefix, consolesLin
     <Navbar className="mb-3" dark color="dark" expand="md" fixed="top">
       <NavbarToggler onClick={toggle} />
       <Link className="pt-0 navbar-brand" to={`${pathPrefix}/new/graph`}>
+        <img src="./prometheus_logo_grey.svg" alt="Prometheus logo" title="Prometheus" />
```
Comment (juliusv) [inline web/ui/react-app/src/Navbar.tsx:28]: Try adding `class="d-inline-block align-top"` (like in https://getbootstrap.com/docs/4.0/components/navbar/#brand) to vertically center the logo with the text.

Comment (juliusv) [reply web/ui/react-app/src/Navbar.tsx:28]: And a `margin-top: 4px;` on the entire Link around it to make the logo+text combo more centered with regards to the other menu items.

**Comment by juliusv** [comment]
@roidelapluie I think I preferred the 4px top margin because then it's more vertically centered in the nav bar, even though the bottom of the "Prometheus" does not 100% align with the nav items then.

**Comment by roidelapluie** [comment]
I have changed a bit the values, can you check now?

**Comment by Nexucis** [comment]
@roidelapluie could you rebase this PR or restart?
It would be cool to merge it !

**Comment by Nexucis** [comment]
closing this PR as it has been replaced by https://github.com/prometheus/prometheus/pull/10233

**Pull request closed without merging**